### PR TITLE
MISUV-8776 Address issues with R&R Debits on Tax Year Summary Page

### PR DESCRIPTION
**Pull request by danielmaltz1**

### CODE REVIEW

File: app/models/financialDetails/ChargeItem.scala [hunk 1]
```diff
@@ -140,11 +140,10 @@ object ChargeItem {
 
   implicit val format: Format[ChargeItem] = Json.format[ChargeItem]
 
-  def filterChargeWithOutstandingAmount(isChargeTypeEnabled:Boolean, chargeType: ChargeType*)
-                  (chargeItem: TransactionItem): Boolean = {
+  def filterAllowedCharges(isChargeTypeEnabled:Boolean, chargeType: ChargeType*)
```
Comment (avelx) [inline app/models/financialDetails/ChargeItem.scala:143]: as I understand we don't have any ITs coverage for filtering?

Comment (danielmaltz1) [reply app/models/financialDetails/ChargeItem.scala:143]: No it doesn't look like it, will add some

File: test/models/financialDetails/ChargeItemSpec.scala [hunk 2]
```diff
@@ -365,4 +366,31 @@ class ChargeItemSpec extends UnitSpec with ChargeConstants  {
         }
       }
   }
+
+  "filterAllowedCharges" should {
+    "filter out FS related charges" when {
+      "relevant FS is disabled" in {
```
Comment (avelx) [inline test/models/financialDetails/ChargeItemSpec.scala:372]: I would suggest to have one UT to cover scenario with empty List

**Review by avelx — DISMISSED** [review]
approved with comments

**Pull request merged**